aws/standard/main.tf

```diff
@@ -1,10 +1,9 @@
 ## we assume a default vpc. if you have one you want to use you will need to provide a vpc and subnet ID
 
 module "redpanda-cluster" {
-  source                   = "redpanda-data/redpanda-cluster/aws"
-  version                  = "~> 0.1"
+  source                   = "git::github.com/redpanda-data/terraform-aws-redpanda-cluster.git?ref=private-ip-and-zones"
```

Comment:
Will `aws/standard` be our new default way to create a redpanda cluster? if that's correct, why reference `private-ip-and-zones` ?

Also, please update the readme so we point our users to the correct aws folder for the quickstart.

Reply:
Oh, you updated this in another commit later. Still, please update the readme with the new path :+1:

Reply:
This is a testing value that will revert to the standard TF module once I've cut a release (which will happen very soon).